CHANGELOG.md

```diff
@@ -21,6 +21,7 @@ to [Semantic Versioning](http://semver.org/).
   new releases of the site.
 - The dev version modal now links to the latest release
   of the site if the current git revision is unavailable.
+- New homepage design
```

Comment:
Also, the new homepage design is a BIG DEAL so you should put it at the very top of the list :dancer: